README.md

```diff
@@ -104,6 +104,7 @@ let config = require('./nuxt.config.js')
 config.dev = (process.env.NODE_ENV !== 'production')
 
 let nuxt = new Nuxt(config)
+await nuxt.ready()
```

Comment:
As this is a top-level code and await is not possible, can you please remove `await` part?

Reply:
Yes I'll do it! Maybe it might be interesting to say in comment that it should be asynchronous if it's not top-level?

Reply:
Yeah. We can also catch errors to prevent Unhandled Promises. For simplicity of readme, i suggest something like `nuxt.ready().catch(console.error)`.

Reply:
Or... Just commenting that ready() returns a promise :)

Reply:
`.catch(console.error)` is self-explanatory, nice 👍🏻